src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -105,9 +105,29 @@
              :else {:success typed})))
    :render
    (fn [{:keys [props state after-update]}]
-     (let [{:keys [editing?]} props]
+     (let [{:keys [editing? writer?]} props]
        [:div {}
-        (style/create-section-header "Workspace Attributes")
+        (style/create-section-header
+          [:div {} "Workspace Attributes"
+           (style/create-inline
```

Comment:
I don't think we need `create-inline` at all.  Instead, just make this a span, with `:fontSize "initial"` to cancel out the header size.